x-pack/plugins/security_solution/public/overview/pages/detection_response.tsx

```diff
@@ -72,12 +101,20 @@ const DetectionResponseComponent = () => {
               <EuiFlexItem>
                 <EuiFlexGroup>
                   <EuiFlexItem>
-                    {hasIndexRead && hasKibanaREAD && <>{'[alerts charts]'}</>}
+                    {hasIndexRead && hasKibanaREAD && (
+                      <EuiFlexGroup>
+                        <AlertsByStatus
+                          title="Alerts"
+                          signalIndexName={signalIndexName}
+                          queryId={DETECTION_RESPONSE_ALERTS_BY_STATUS_ID}
+                          detailsButtonOptions={detailsButtonOptions}
```

Comment:
Why is the `title`, `queryId` and `detailsButtonOptions` implemented up here in the main page and passed down? I don't see them being used anywhere, rather than inside the AlertsByStatus component, or other deeply nested components.

We should always place the definition/implementation of the parts as close as we can to the place they are used. The more properties we have in our components the more coupled they become, which harms maintainability and readability. 
We have a lot of components, we should try to keep them as loosely coupled (independent) and highly cohesive (single task) as possible, to ensure we can maintain them easily in the long term.